src/constructs/lambda/lambda.ts

```diff
@@ -33,5 +33,7 @@ export class GuLambdaFunction extends Function {
         enabled: true,
       });
     });
+
+    bucket.grantRead(this);
```

Comment:
Is this bucket the same one accounts use for their EC2 based artefacts? If so, we should be able to re-use [this construct](https://github.com/guardian/cdk/blob/main/src/constructs/iam/policies.ts#L96) or even create a `GuGetDistributable` construct.